requirements.in

```diff
@@ -27,3 +27,5 @@ tldextract
 urlextract
 uvicorn
 wheel
+deepdiff
+rich
```

Comment:
they result in a lot additional dependencies. Do we really need them? Or is it possible to come around them?

Reply:
I have removed rich now and made use of the already present colorama dependency. Deepdiff is still needed though as it does a lot of the comparison. I think this library will also allow us to revise the the `auto_rule_tester`.